tests/layer_tests/mo_python_api_tests/test_mo_convert_import_from_memory.py

```diff
@@ -0,0 +1,475 @@
+    def f(x):
+        return keras_net(x)
+
+    shape = PartialShape([-1, 1, 2, 3])
+    param1 = ov.opset8.parameter(shape, name="x:0", dtype=np.float32)
+    param2 = ov.opset8.parameter(shape, name="x_1:0", dtype=np.float32)
+    add = ov.opset8.add(param1, param2)
+    add.get_output_tensor(0).set_names({"model/tf_op_layer_add/add:0"})
+    relu = ov.opset8.relu(add)
+    relu.get_output_tensor(0).set_names({"model/tf_op_layer_Relu/Relu:0"})
+    sigm = ov.opset8.sigmoid(relu)
+    sigm.get_output_tensor(0).set_names({"model/tf_op_layer_Sigmoid/Sigmoid:0", "Identity:0"})
+
+    parameter_list = [param1, param2]
+    model_ref = Model([sigm], parameter_list, "test")
+
+    return keras_net, model_ref, None
+
+
+def create_tf_saved_model(temp_dir):
+    import tensorflow as tf
+
+    input_names = ["Input1", "Input2"]
+    input_shape = [1, 2, 3]
+
+    x1 = tf.keras.Input(shape=input_shape, name=input_names[0])
+    x2 = tf.keras.Input(shape=input_shape, name=input_names[1])
+    y = tf.nn.sigmoid(tf.nn.relu(x1 + x2))
+    keras_net = tf.keras.Model(inputs=[x1, x2], outputs=[y])
+
+    shape = PartialShape([-1, 1, 2, 3])
+    param1 = ov.opset8.parameter(shape, name="Input1:0", dtype=np.float32)
+    param1.get_output_tensor(0).set_names({"input1:0", "Func/PartitionedCall/input/_0:0"})
+    param2 = ov.opset8.parameter(shape, name="Input2:0", dtype=np.float32)
+    param2.get_output_tensor(0).set_names({"input2:0", "Func/PartitionedCall/input/_1:0"})
+    add = ov.opset8.add(param1, param2)
+    add.get_output_tensor(0).set_names({"PartitionedCall/model/tf_op_layer_add/add:0"})
+    relu = ov.opset8.relu(add)
+    relu.get_output_tensor(0).set_names({"PartitionedCall/model/tf_op_layer_Relu/Relu:0"})
+    sigm = ov.opset8.sigmoid(relu)
+    sigm.get_output_tensor(0).set_names({"PartitionedCall/model/tf_op_layer_Sigmoid/Sigmoid:0",
+                                         "PartitionedCall/Identity:0",
+                                         "Identity:0",
+                                         "Func/PartitionedCall/output/_2:0"})
+
+    parameter_list = [param1, param2]
+    model_ref = Model([sigm], parameter_list, "test")
+
+    tf.saved_model.save(keras_net, temp_dir + "/model")
+    saved_model = tf.saved_model.load(temp_dir + "/model")
+
+    return saved_model, model_ref, None
+
+
+class TestImportFromMemory(CommonMOConvertTest):
+    test_data = [
+        # PyTorch
+        create_pytorch_nn_module_case1,
+        create_pytorch_nn_module_case2,
+        create_pytorch_nn_module_case3,
+        create_pytorch_jit_script_module,
+        create_pytorch_jit_script_function,
+
+        # TF2
+        create_keras_model,
+        create_keras_layer,
+        create_tf_function,
+        create_tf_module,
+        create_tf_checkpoint,
+        create_tf_saved_model,
+
+        # TF1
+        create_tf_graph_def,
+        create_tf1_wrap_function,
+        create_tf_session,
+    ]
```

Comment:
Please split it to have separate file and suite for each framework.
Please add `@pytest.mark.precommit_tf_fe` for TF FE pre-commit.
Double-check that this really runs in the pre-commit.

Reply:
Done.